demo/Snakefile

```diff
@@ -34,7 +34,7 @@ configfile: "../modules/sequenza/1.4/config/default.yaml"
 configfile: "../modules/strelka/1.1/config/default.yaml"
 configfile: "../modules/utils/2.0/config/default.yaml"
```

Comment:
Can you remove liftOver from the demo and sync this PR up with Master?

Reply:
Just did it